cl/corpus_importer/tasks.py

```diff
@@ -1203,21 +1220,228 @@ def make_docket_by_iquery(
     d = async_to_sync(find_docket_object)(
         court_id,
         str(pacer_case_id),
-        report.data["docket_number"],
+        report_data["docket_number"],
         using=using,
     )
 
     d.pacer_case_id = pacer_case_id
     d.add_recap_source()
     return save_iquery_to_docket(
         self,
-        report.data,
+        report_data,
         d,
         tag_names,
         add_to_solr=True,
+        avoid_trigger_signal=avoid_trigger_signal,
     )
 
 
+# Retry 10 times. First one after 1m, then again every 5 minutes.
+@app.task(
+    bind=True,
+    autoretry_for=(PacerLoginException, RedisConnectionError),
+    max_retries=10,
+    interval_start=1 * 60,
+    interval_step=5 * 60,
+    ignore_result=True,
+)
+def make_docket_by_iquery(
+    self: Task,
+    court_id: str,
+    pacer_case_id: int,
+    using: str = "default",
+    tag_names: Optional[List[str]] = None,
```

Comment:
We can refactor this as:

```suggestion
    tag_names: List[str] | None = None,
```